APIBlueprintGenerator.coffee

```diff
@@ -88,6 +88,47 @@ APIBlueprintGenerator = ->
         description: description,
       }
 
+  # Generate a parameter List for the mustache context from a paw Request
+  #
+  # @param [Request] exchange The paw HTTP request
+  #
+  # @return [Object] The template context object
+  #
+  @parameter = (paw_request) ->
+    parameters = []
+    body = paw_request.body
+    has_body = body.length > 0
+    url_parameters = paw_request.getUrlParameters()
+    for key, value of url_parameters
+      parameters.push({
+        name: key,
+        example: value,
+        type: @isNumber(value)
+        })
+    if has_body
```

Comment:
Sorry for not getting back to you sooner.

I was suggesting that you only handle the parameters section and remove the attributes logic from this PR.

However, I am happy for you to support attributes, it might be a bit more work to handle it correctly. In the long run we would like to solve 1 (#14) which would achieve this.

Thanks for the work here.